website/scripts/generate-dynamic-pages/generate-pages-from-registry.ts

```diff
@@ -123,17 +132,10 @@ const generateNodePages = (
     writeFileSync(
       filePath,
       nodeTemplate(
-        factory(
-          desc,
-          {
-            customEvents: {},
-            variables: {}
-          } as any,
-          {
-            numInputs: 2,
-            numOutputs: 2
-          }
-        ),
+        nodeFactory(graph, {
```

Comment:
is this still working for nodes with dynamic i/o? I wasn't able to render the inputs without it

Reply:
I didnt check this - my main goal was to get this to compile and the scripts to run - but if you want to attempt or suggest a fix glad to take a look.

Reply:
we can check it after the merge